src/cmd/turnkey/pkg/wallets.go

```diff
@@ -19,11 +20,18 @@ var (
 )
 
 func init() {
-	walletCreateCmd.Flags().StringVar(&walletNameOrID, "name", "", "name to be applied to the wallet")
+	walletCreateCmd.Flags().StringVar(&walletNameOrID, "name", "", "name to be applied to the wallet.")
+
+	walletInitImportCmd.Flags().StringVar(&user, "user", "", "ID of user to importing the wallet")
+	walletInitImportCmd.Flags().StringVar(&importBundlePath, "import-bundle-output", "", "filepath to write the import bundle to.")
+
+	walletImportCmd.Flags().StringVar(&user, "user", "", "ID of user to importing the wallet")
```

Comment:
While these two commands should never be executed on the same run, it feels dangerous to reuse the variables like this